Algorithms/check_permutation.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# CtCI 6th Edition Problem 1.2
+# Given two strings, write a method to decide if one is a permutation of the
+# other.
+
+from timeit import timeit
+import collections
+
+
+def is_permutation(string1, string2):
+    return (len(string1) == len(string2)
+            and sorted(string1) == sorted(string2))
```

Comment:
This is pure style but:
1. `and` should probably be on the previous line so that you know it continues to the next line
2. You can probably make this one line. It's not super long.